test/playlists.test.js

```diff
@@ -0,0 +1,68 @@
+import QUnit from 'qunit';
+import { makeMochTech } from './util/util.js';
+import sinon from 'sinon';
+import { FlashlsHandler } from '../src/index.js';
+
+QUnit.module('Flashls playlists', {
+  beforeEach() {
+    this.clock = sinon.useFakeTimers();
+  },
+  afterEach() {
+    this.clock.restore();
+  }
+});
+
+QUnit.test('triggers an event when the active media changes', function(assert) {
```

Comment:
I think it would be good to break this test up into two different tests, one that just tests the `mediachange` event, and one that just tests the `playlists.media` function